app/actions/local/category.ts

```diff
@@ -25,6 +50,22 @@ export const storeCategories = async (serverUrl: string, categories: CategoryWit
     const models = await Promise.all(modelPromises);
     const flattenedModels = models.flat() as Model[];
 
+    if (prune && categories.length) {
```

Comment:
Probably cannot happen, but what if there are no categories at all? If there are no categories at all, and we try to prune (i.e. remove all categories) we won't be able.

Reply:
I think there always will be 3 categories for a user in a team; DMs + Channels + Favourites. I'll add it to the next PR to never delete those 3 and that way we can just call prune without categories as well to remove all.